mongodb/configure.sls

```diff
@@ -16,6 +16,13 @@ place_mongodb_config_file:
     - watch_in:
       - service: mongodb_service_running
 
+check_systemd_override_directory:
+  file.directory:
+    - name: /etc/systemd/system/multi-user.target.wants/mongodb.service.d
+    - user: root
+    - group: root
+    - dir_mode: 755
+
```

Comment:
You can replace this with just appending `makedirs: True` to the file.managed state for the unit file.

Reply:
OK, that's better!